Peekabook/Peekabook/Presentation/AddBook/VC/BarcodeVC.swift

```diff
@@ -75,22 +75,20 @@ extension BarcodeVC {
         }
     }
     
-    private func fetchBooks() {
-        let ls = NaverSearchAPI.shared
-        ls.getNaverSearchData(d_titl: "", d_isbn: "\(isbnCode)", display: displayCount) { [weak self] result in
-            if let result = result {
-                self?.bookInfoList = [BookInfoModel(image: "", title: "", author: "")]
-                DispatchQueue.main.async {
-                    let nextVC = AddBookVC()
-                    nextVC.bookInfo = [BookInfoModel(image: "", title: "", author: "")]
+    private func getNaverSearchData(d_titl: String, d_isbn: String, display: Int) {
+        NaverSearchAPI.shared.getNaverSearchData(d_titl: d_titl, d_isbn: d_isbn, display: display) { response in
+            if let response = response {
+                let nextVC = AddBookVC()
+                nextVC.searchType = 1
+                self.bookInfoList = [BookInfoModel(image: "", title: "", author: "")]
+                
+                if [BookInfoModel(image: "", title: "", author: "")].isEmpty {
+                    self.showErrorPopUp()
+                } else {
+                    nextVC.bookInfo = [BookInfoModel(image: response[0].image, title: response[0].title, author: response[0].author)]
+                    nextVC.dataBind(model: BookInfoModel(image: response[0].image, title: response[0].title, author: response[0].author))
```

Comment:
p3;
혹시 response[0]으로 하드코딩 하다 오류가 날 수 있으니 이런식으로 바꿔볼 수도 있겠네요~
```suggestion
                    let info = response[0]
                     nextVC.bookInfo = [BookInfoModel(image: info.image, title: info.title, author: info.author)]
                     nextVC.dataBind(model: BookInfoModel(image: info.image, title: info.title, author: info.author))
```